CHANGES.md

```
@@ -235,9 +256,29 @@ To be released.
 [#70]: https://github.com/dahlia/dojang/pull/70
 [#71]: https://github.com/dahlia/dojang/pull/71
 [#72]: https://github.com/dahlia/dojang/pull/72
+[#73]: https://github.com/dahlia/dojang/pull/73
 
 ### Haskell API
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)